pkg/jobrunaggregator/jobtableprimer/generate_jobnames.go

```diff
@@ -0,0 +1,190 @@
+package jobtableprimer
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorlib"
+)
+
+type generateJobNamesFlags struct {
+	periodicURLs      []string
+	releaseConfigURLs []string
+}
+
+func newGenerateJobNamesFlags() *generateJobNamesFlags {
+	return &generateJobNamesFlags{
+		periodicURLs: []string{
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml",
+		},
+		releaseConfigURLs: []string{
+			"https://raw.githubusercontent.com/openshift/release/master/core-services/release-controller/_releases/release-ocp-4.10-ci.json",
+			"https://raw.githubusercontent.com/openshift/release/master/core-services/release-controller/_releases/release-ocp-4.10.json",
+		},
```

Comment:
Could we have the releases as a flag instead, and automatically generate these URI's? It'd make it easier to update